simulation/regen/basic_regen.py

```diff
@@ -1,10 +1,45 @@
-from simulation.regen.base_regen import BaseRegen
+from simulation.regen import BaseRegen
+import numpy as np
 
 
 class BasicRegen(BaseRegen):
+
     def __init__(self):
         super().__init__()
+        self.efficiency = 0.5   # currently set to 50% but best case scenario is 60-70%
+        self.min_decel_mag = 0
+        self.vehicle_mass = 250
+        self.kmk_to_mps = 0.278
```

Comment:
nit: typo

Reply:
Might be better to have this as a global const for the file rather than an instance variable